### DISCUSSION TITLE
airdrop

### DISCUSSION THREAD

**Discussion by mohammadrahbarii** [zkSync-Community-Hub/zksync-developers#592] [ZK Stack]
### Module Affected

- [ ] System contracts (era-system-contracts)
- [ ] Smart contracts (era-contracts)
- [ ] zkEVM (era-zk_evm)
- [ ] Server (zksync-era)
- [ ] Prover (era-boojum)
- [ ] Fee withdrawer (era-fee-withdrawer)
- [ ] Block Explorer (clock-explorer)
- [ ] Portal - Wallet + Bridge (dapp-portal)
- [X] Other (please specify below)

### Rust Version

private

### Operating System

- [ ] Ubuntu
- [ ] MacOS
- [ ] Windows
- [X] Other (Please specify below)

### Issue Description

Hello I have been your user for more than 3 years, since the test net. I tried to have a few transactions (bridge, swap, and mint) every month, and I participated in almost all campaigns. But the Airdrop article did not belong to me. This means that my right has been violated. I only had and still have one Volt, you can check it completely. I want you to reconsider, and support your active user.    Well, you could do a form to make it easy for anyone to do so ... maybe not mistakenly distributed for many people like me         0xde1Bba6fE23e7D83eb4028f84Bd1daFcef05a43C   

### Expected Behavior

fair distribution

### Current Behavior

Unfair distribution

### Prior Research

- [X] Yes, I searched for similar issues and did not find any matching ones.

**Comment by 0xsisyfos** [comment]
Hello,

Thank you for reaching out.

The ZK Token FAQ provides additional details on eligibility criteria: https://docs.zknation.io/zk-token/zk-token-faq

In particular, question 3 ("I met some of the eligibility criteria, but did not get an allocation, why not?") answers your question directly.

As a note, this space if for developer questions, please join the Discord for further support: https://join.zksync.dev/
